### PR TITLE
add komfovent integration docs

### PR DESCRIPTION
**Pull request by ProstoSanja**
## Proposed change

Adding documentation about new Komfovent integration to go in pair with the core PR.  

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [x] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [x] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/95722
- Link to parent pull request in the Brands repository: https://github.com/home-assistant/brands/pull/4510
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/komfovent.markdown [hunk 1]
```diff
@@ -0,0 +1,34 @@
+---
+title: Komfovent
+description: Instructions on how to integrate Komfovent ventilation units into Home Assistant.
+ha_category:
+  - Climate
+ha_release: 2023.7
```
Comment (joostlek) [inline source/_integrations/komfovent.markdown]:
```suggestion
ha_release: 2023.12
```

File: source/_integrations/komfovent.markdown [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+ha_platforms:
+  - climate
+ha_integration_type: integration
+---
+
+Integrates Komfovent ventilation units into Home Assistant.
+
+This integration is **fully local** and does not utilize the Komfovent Control cloud interface.
+
+For integration to work your unit has to be connected to your local network via the ethernet cable, as most Komfovent units do not support wireless connectivity. After that you can check the IP address of the unit on the control panel by long pressing the "Settings" button.
```
Comment (c0ffeeca7) [inline source/_integrations/komfovent.markdown]:
```suggestion
For the integration to work, your unit has to be connected to your local network via Ethernet cable. Most Komfovent units do not support wireless connectivity. Once you have an Ethernet connection, you can check the IP address of the unit on the control panel by long pressing the **Settings** button.  
```

File: source/_integrations/komfovent.markdown [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+ha_integration_type: integration
+---
+
+Integrates Komfovent ventilation units into Home Assistant.
+
+This integration is **fully local** and does not utilize the Komfovent Control cloud interface.
+
+For integration to work your unit has to be connected to your local network via the ethernet cable, as most Komfovent units do not support wireless connectivity. After that you can check the IP address of the unit on the control panel by long pressing the "Settings" button.  
+  
+For integration you will need the IP address, username ("user" is the default one) and password ("user" is once again the default).
```
Comment (c0ffeeca7) [inline source/_integrations/komfovent.markdown]:
```suggestion
When adding the integration in Home Assistant, you will need the IP address, the username ("user" is the default one) and password ("user" is once again the default).  
```

File: source/_integrations/komfovent.markdown [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+For integration you will need the IP address, username ("user" is the default one) and password ("user" is once again the default).  
+
+This integration currently supports the following features:
+
+- Getting the current operating mode (AWAY, NORMAL, INTENSIVE, BOOST)
+- Getting the current room temperature
+- Setting the operating mode
+  
+### N.B: Password!  
+It is highly suggested you change the password before integrating the unit with the Home Assistant. For that navigate to the IP address of the unit through a web browser and change the password in settings. Beware, password has a **max size of 10 characters** in Komfovent units, so if you put in a bigger string, it will truncate it, but it does not do so when you try and login, which may lead you to believe you forgot your password or something is not working.
```
Comment (c0ffeeca7) [inline source/_integrations/komfovent.markdown]:
```suggestion
It is highly suggested you change the password before integrating the unit with the Home Assistant. 
1. To access the webinterface of the device, in the browser, enter the IP address of the unit.
2. Change the password in the settings. 
    - The password has a **max size of 10 characters** in Komfovent units. Longer passwords are truncated here, but not on the login screen.
    -  When you try and fail at login, before recovering the password, check if your password is longer than 10 characters. Drop all characters after 10 and try again.
```

**Review by c0ffeeca7 — COMMENTED** [review]
Tweaks

File: source/_integrations/komfovent.markdown [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+When adding the integration in Home Assistant, you will need the IP address, the username ("user" is the default one) and password ("user" is once again the default).  
+
+This integration currently supports the following features:
+
+- Getting the current operating mode (AWAY, NORMAL, INTENSIVE, BOOST)
+- Getting the current room temperature
+- Setting the operating mode
+  
+### N.B: Password!  
+It is highly suggested you change the password before integrating the unit with the Home Assistant.
```
Comment (c0ffeeca7) [inline source/_integrations/komfovent.markdown]:
```suggestion
It is highly suggested you change the password before integrating the unit with Home Assistant. 
```

File: source/_integrations/komfovent.markdown [hunk 6]
```diff
@@ -0,0 +1,38 @@
...
+
+This integration currently supports the following features:
+
+- Getting the current operating mode (AWAY, NORMAL, INTENSIVE, BOOST)
+- Getting the current room temperature
+- Setting the operating mode
+  
+### N.B: Password!  
+It is highly suggested you change the password before integrating the unit with the Home Assistant. 
+1. To access the webinterface of the device, in the browser, enter the IP address of the unit.
```
Comment (c0ffeeca7) [inline source/_integrations/komfovent.markdown]:
```suggestion
1. To access the device's web interface, enter the unit's IP address in the browser.
```

File: source/_integrations/komfovent.markdown [hunk 7]
```diff
@@ -0,0 +1,38 @@
...
+2. Change the password in the settings. 
+    - The password has a **max size of 10 characters** in Komfovent units. Longer passwords are truncated here, but not on the login screen.
```
Comment (c0ffeeca7) [inline source/_integrations/komfovent.markdown]:
```suggestion
    - The password has a **maximum size of 10 characters** in Komfovent units. Longer passwords are truncated here but not on the login screen.
```

File: source/_integrations/komfovent.markdown [hunk 8]
```diff
@@ -0,0 +1,38 @@
...
+    -  When you try and fail at login, before recovering the password, check if your password is longer than 10 characters. Drop all characters after 10 and try again.
```
Comment (c0ffeeca7) [inline source/_integrations/komfovent.markdown]:
```suggestion
    -  When you try and fail to login, before recovering the password, check if your password is longer than 10 characters. Drop all characters after 10 and try again.
```

**Review by c0ffeeca7 — APPROVED** [review]
Thank you, @ProstoSanja :+1:

**Pull request merged**